composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": ">=7.2",
-        "guzzlehttp/guzzle": "^6.5",
+        "guzzlehttp/guzzle": "^7.0",
```

Comment:
Is it possible to support both versions? Guessing a lot of people will still be on 6.x

Reply:
I think we can do that, the same you did to `illuminate/notification` and `illuminate\support`:

> "illuminate/notifications": "~5.5 || ~6.0 || ~7.0"
> "illuminate/support": "~5.5 || ~6.0 || ~7.0"

Would you like me to make another pull request of that?

Reply:
Sure, just update the PR.

Reply:
Check: https://github.com/guzzle/guzzle/blob/master/UPGRADING.md